bin/porta

```diff
@@ -569,19 +592,22 @@ module Porta
 
     def run
       queues_arg = queues.map { |queue| "--queue #{queue}" }
-      exec_in_porta(envs) { "bundle exec sidekiq #{queues_arg.join(' ')} -c #{concurrency}" }
+      exec_in_porta(envs) { "#{'NLS_LANG=AMERICAN_AMERICA.UTF8'} bundle exec sidekiq #{queues_arg.join(' ')} -c #{concurrency}" }
```

Comment:
`#{'NLS_LANG=AMERICAN_AMERICA.UTF8'}` really needed here?

Reply:
This is probably for better Oracle support. Maybe needs to go to `server` as well. Now sure Jakub will be interested in finalizing this PR. Perhaps I will try and look at it when I have some more time.

Reply:
Actually this is in a porta initializer now, so should be redundant. (note to anybody looking at this in the future)